deviceregistrationpolicy/device_registration_policy_request_builder.go

```diff
@@ -1,130 +0,0 @@
-package deviceregistrationpolicy
```

Comment:
Question: Do we intend to treat deprecation of entire functionality as a major change?

Reply:
not under beta, breaking changes are ok, this is what beta APIs are for.